blocks/product-list-page/product-list-page.js

```diff
@@ -2,14 +2,21 @@ import ProductList from '@dropins/storefront-product-discovery/containers/Produc
 import Facets from '@dropins/storefront-product-discovery/containers/Facets.js';
 import ResultsInfo from '@dropins/storefront-product-discovery/containers/ResultsInfo.js';
 import { render as provider } from '@dropins/storefront-product-discovery/render.js';
+import { Button, Icon, provider as UI } from '@dropins/tools/components.js';
+// Wishlist Dropin
+import { WishlistToggle } from '@dropins/storefront-wishlist/containers/WishlistToggle.js';
+import { render as wishlistRender } from '@dropins/storefront-wishlist/render.js';
+// Cart Dropin
+import * as cartApi from '@dropins/storefront-cart/api.js';
 import { rootLink } from '../../scripts/scripts.js';
-// import { readBlockConfig } from '../../scripts/aem.js';
+import { readBlockConfig } from '../../scripts/aem.js';
 
 // Initializers
 import '../../scripts/initializers/search.js';
```

Comment:
Are we double initializing `search`? I thought PLP would get it's own initializer with the dropin.